content/english/post/living_light.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Living_light"
+date: 2023-03-24T15:15:38-05:00
+Description: "This is my first post about my ongoing personal project living_light. In this post I will introduce the project and describe my intentions for creating it."
+Tags: []
+Categories: []
+DisableComments: false
+draft: true
+thumbnail: "/images/living_light_front.jpeg"
+---
+
+This is my first post about my ongoing personal project living_light. In this post I will introduce the project and describe my intentions for creating it.
+
+## What is living_light
+
+living_light is a decorative lamp that slowly rises and falls in brightness at the rate of meditative breathing. The color and brightness follows the day's natural progression and our circadian rhythms. During the day it is a bright blueish white and then at sunset it transitions to a dark red for the night. It is an open invitation to pause to take a breath and a reminder follow the natural solar cycle.
+
+Currently the light has a very limited user interface. I had spent a lot of time implementing a text interface to change the behavior of the light through Telegram messenger's bot API that would take simple commands to change the length of the breath cycle, the color, and brightness. I found that having these options changed the way I engaged with the light. With these options, I found myself distracted by assessing whether or not I should change the settings instead of meditating with it. Having these options detracted from the experience I designed, so I removed them.
```

Comment:
```suggestion
Currently the light intentionally has a very limited user interface. I had spent a lot of time implementing a text interface to change the behavior of the light through Telegram messenger's bot API that would take simple commands to change the length of the breath cycle, the color, and brightness. However, I found that having these options changed the way I engaged with the light; I found myself distracted by assessing whether or not I should change the settings instead of meditating with it. Having these options detracted from the experience I designed, so I removed them. [Perhaps add a sentence to wrap up by talking about how the experience is now without those options.]
```